app/views/users/new.html.haml

```diff
@@ -14,6 +14,10 @@
             .col-sm-6
               = f.password_field :password, hide_label: true
           .form-group
+            %label.control-label.col-sm-2 Confirm Password
+            .col-sm-6
+              = f.password_field :password_confirmation, hide_label: true
```

Comment:
How come you are manually creating the labels and hiding the automatic ones? If it is because of the widths, you should remove all of your `form-group`s and `col` divs and just set the widths on the form options.

```haml
           .form-group
             %label.control-label.col-sm-2 Confirm Password
             .col-sm-6
               = f.password_field :password_confirmation, hide_label: true
```

should be:

```haml
= f.password_field :password_confirmation
```